mlflow/tracing/fluent.py

```diff
@@ -330,6 +330,7 @@ def get_trace(request_id: str) -> Optional[Trace]:
 @experimental
 def search_traces(
     experiment_ids: Optional[List[str]] = None,
+    run_id: Optional[str] = None,
```

Comment:
Can we add this parameter at the end for backwards compatibility? Otherwise, this could break existing workflows. In general, we should get in the habit of writing methods with keyword-only args (whoops, my fault) to avoid having to think about this in the future

Reply:
Ah that totally makes sense, will update the order!